services/system/Transact/src/RTransact.cpp

```diff
@@ -584,7 +584,6 @@ std::optional<HttpReply> RTransact::serveSys(const psibase::HttpRequest& request
       auto data     = psio::from_frac<LoginData>(loginAct.rawData());
       if (checkExp(trx.transaction->tapos().expiration()) && data.rootHost == request.rootHost)
       {
-         auto appHost = app.str() + "." + request.rootHost;
```

Comment:
This makes it so app is never used anywhere. I thought the token should be derived from both app & rootHost.